.php_cs.dist

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$finder = Symfony\Component\Finder\Finder::create()
+    ->notPath('vendor')
+    ->notPath('bootstrap')
+    ->notPath('storage')
+    ->in(__DIR__)
+    ->name('*.php')
+    ->notName('*.blade.php');
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'ordered_imports' => ['sortAlgorithm' => 'alpha'],
+        'no_unused_imports' => true,
+
+        // PSR-12
+        'blank_line_after_opening_tag' => true,
+        'braces' => ['allow_single_line_closure' => true],
+        'compact_nullable_typehint' => true,
+        'concat_space' => ['spacing' => 'one'],
+        'declare_equal_normalize' => ['space' => 'none'],
+        'function_typehint_space' => true,
+        'new_with_braces' => true,
+        'method_argument_space' => ['on_multiline' => 'ensure_fully_multiline'],
+        'no_empty_statement' => true,
+        'no_leading_import_slash' => true,
+        'no_leading_namespace_whitespace' => true,
+        'no_whitespace_in_blank_line' => true,
+        'return_type_declaration' => ['space_before' => 'none'],
+        'single_trait_insert_per_statement' => true,
```

Comment:
```suggestion
        'single_trait_insert_per_statement' => true,
        'no_extra_blank_lines' => [
            'curly_brace_block',
            'extra',
            'parenthesis_brace_block',
            'throw',
            'use',
        ],
        'ternary_operator_spaces' => true,
        'not_operator_with_successor_space' => true,
        'single_blank_line_before_namespace' => true,
```

Just did a comparison against my config and I have these additional options which may be useful?

Reply:
Done, thanks!